tests/mail/tests.py

```diff
@@ -132,6 +132,22 @@ def test_cc(self):
             ['to@example.com', 'other@example.com', 'cc@example.com', 'cc.other@example.com', 'bcc@example.com']
         )
 
+    def test_cc_headers(self):
+        email = EmailMessage(
+            'Subject', 'Content', 'bounce@example.com', ['to@example.com'],
+            cc=['foo@example.com'], headers={'Cc': 'override@example.com'},
+        )
+        message = email.message()
```

Comment:
Perhaps `.message()` could be done on the previous line after the ) since the `email` variable is otherwise unused.